### ISSUE TITLE
Add support for Kotlin files

### ISSUE THREAD

**Issue by antonyalkmim** [Malinskiy/danger-jacoco#2]
This plugin is currently only compatible with Java files. It would be very helpful to add support for Kotlin files as well.

This line select only java files: 
```ruby 
affected_files.select { |file| file.end_with? '.java' }
```
it would be nice to select `.kt` files too.

And on `extract_class ` method, it removes the `.java` string from filename with the following instruction, so for Kotlin files, it returns an incorrect file name.
```ruby
file[0, file.length - 5].split(java_path_delimiter)[1]
```

**Comment by antonyalkmim** [comment]
I need to use this plugin on my kotlin project, so I will work on these changes on my fork and open an PR. 👍

**Comment by antonyalkmim** [comment]
This feature was implemented in #3 🍻
